<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMachine</class>
 <widget class="QDialog" name="DialogMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Machine</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #F5F0E8;
}

QPushButton {
    background-color: #6B4423;
    color: #FAF8F3;
    border: none;
    padding: 10px 20px;
    border-radius: 5px;
    font-weight: 600;
    font-size: 10pt;
    min-height: 30px;
}

QPushButton:hover {
    background-color: #8B6F47;
}

QGroupBox {
    background-color: #FAF8F3;
    border: 2px solid #C9B896;
    border-radius: 8px;
    margin-top: 15px;
    padding-top: 15px;
    font-weight: 600;
    font-size: 11pt;
    color: #6B4423;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5px 15px;
    background-color: #6B4423;
    color: #FAF8F3;
    border-radius: 4px;
    left: 10px;
}

QLabel {
    color: #5A3820;
    font-size: 10pt;
}

QLineEdit, QComboBox, QSpinBox, QDateEdit {
    background-color: #FFFFFF;
    border: 2px solid #C9B896;
    border-radius: 5px;
    padding: 8px;
    font-size: 10pt;
}

QLineEdit:focus, QComboBox:focus, QSpinBox:focus, QDateEdit:focus {
    border: 2px solid #8B6F47;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Informations Machine</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID Machine :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditID">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Auto-généré</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Référence :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditReference">
        <property name="placeholderText">
         <string>Ex: SCE-001</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditNom">
        <property name="placeholderText">
         <string>Ex: Scie circulaire principale</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Type :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxType">
        <item>
         <property name="text">
          <string>Scie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Perceuse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Raboteuse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ponceuse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Défonceuse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tour à bois</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>État :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxEtat">
        <item>
         <property name="text">
          <string>Disponible</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>En maintenance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hors service</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Date Dernière Maintenance :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDateEdit" name="dateEditMaintenance">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Heures Cumulées :</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="spinBoxHeures">
        <property name="suffix">
         <string> h</string>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Quantité :</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="spinBoxQuantite">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Responsable Maintenance :</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEditRespMaintenance">
        <property name="placeholderText">
         <string>Nom du responsable</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Responsable Atelier :</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEditRespAtelier">
        <property name="placeholderText">
         <string>Nom du responsable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogMachine</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogMachine</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>